<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widgetItem" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>501</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>451</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>455</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border: 2 solid #A5D6A7;
border-radius: 6px;</string>
   </property>
   <widget class="QFrame" name="frameItem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>481</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGraphicsView" name="viewItem">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 6px;
border: 2 solid rgb(255, 170, 127);
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNameItem">
       <property name="styleSheet">
        <string notr="true">color: rgb(125, 92, 73);
font-size: 16px;
font-weight: bold;
border: none;

</string>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSortItem">
       <property name="styleSheet">
        <string notr="true">color: rgb(125, 92, 73);
font-size: 14px;
border: none;
</string>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSortItem_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 170, 127);
font-size: 12px;
border: none;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Watering</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameWateringItem">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lastWateringItem">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 170, 127);
font-size: 12px;
border: none;
</string>
          </property>
          <property name="text">
           <string>lastWatering</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 170, 127);
font-size: 12px;
border: none;
</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nextWateringItem">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 170, 127);
font-size: 12px;
border: none;
</string>
          </property>
          <property name="text">
           <string>nextWatering</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressWateringItem">
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSortItem_3">
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 170, 127);
font-size: 12px;
border: none;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Feritilizer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameFerilizerItem">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lastFeritilizerItem">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 170, 127);
font-size: 12px;
border: none;
</string>
          </property>
          <property name="text">
           <string>lastFeritilizer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 170, 127);
font-size: 12px;
border: none;
</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nextFeritilizerItem">
          <property name="styleSheet">
           <string notr="true">color: rgb(170, 170, 127);
font-size: 12px;
border: none;
</string>
          </property>
          <property name="text">
           <string>nextFeritilizer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressFeritilizerItem">
       <property name="styleSheet">
        <string notr="true">color: white;
</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMoreItem">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);
color: rgb(255, 255, 255);
border: none;
 border-radius: 10px;
 padding: 10px 20px;
font-size: 16px;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>More</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
